modules/sso/jetpack-sso-login.js

```diff
@@ -0,0 +1,25 @@
+jQuery( document ).ready( function( $ ) {
+	var ssoWrap = $( '.jetpack-sso-wrap' ),
```

Comment:
Do we expect to have more the one jetpack-sso-wrap div? Can be make this a id. The same goes for the `a.jetpack-sso-button`